modules/dnn/src/layers/fast_convolution/depthwise_convolution.cpp

```diff
@@ -0,0 +1,381 @@
+/*
+    This file is a part of ficus language project.
+    See ficus/LICENSE for the licensing terms
+*/
+// This file is modified from the ficus (https://github.com/vpisarev/ficus/blob/master/lib/NN/OpConv.fx)
```

Comment:
@vpisarev Please verify as this integration contradicts to 3rdparty original files and/or 3rdparty adopted files.

Reply:
Thanks for code reviewing. Any advice on this? I don't know how to modify it.

Reply:
I expect header similar to modules/dnn/src/layers/fast_convolution/winograd_3x3s1_f63.cpp
where we have OpenCV header on top and then the original license header of the adapted code.

Reply:
Done.